### DISCUSSION TITLE
How to disable servicelb / traefik after installation

### DISCUSSION THREAD

**Discussion by RamazanKara** [kairos-io/kairos#1662] [General]
the ```- --disable=servicelb``` flag in /oem 90-custom-config.yaml is not working after installation. Servicelb is still deployed. Do I have to reinstall?

**Comment by mevatron** [comment]
I actually have the opposite of this problem, I actually disabled both services with `--disable=traefik,servicelb`, and was wondering how I can re-enabled them on the fly without reinstall?

I have the [system-upgrade-controller](https://kairos.io/docs/advanced/after-install/#with-kubernetes) installed as specified, and was curious if it's possible to just remove the `--disable` with a new config? I see the example for a custom swapfile addition in the above. Can someone show how to modify the existing k3s config please?

**Reply by mevatron** [reply]
I also found the documentation on performing a [reset](https://kairos.io/docs/reference/reset/#from-kubernetes) via Kubernetes. Would I just pass in the original cloud-config with the new k3s switches I would prefer here? Is the reset required for these changes?
